posthog/api/dashboard.py

```diff
@@ -203,7 +203,7 @@ def get_result(self, dashboard_item: DashboardItem):
         result = cache.get(dashboard_item.filters_hash)
         if not result or result.get("task_id", None):
             return None
-        return result["result"]
+        return result.get("data")
```

Comment:
Just a note, I think this might bust the items in the cache currently

Reply:
Yep, I sort of thought it was worth it but maybe i should use result instead of data to not lose any data?